app/models/mapping.rb

```diff
@@ -28,11 +28,17 @@ def current
         end
       end
 
+  attr_accessible :locid
   attr_accessible :iri, :source, :target, :kind, :theorem, :proven, :local,
                   :inclusion, :logic_mapping, :parent, :ontology_id, :source_id,
                   :target_id, :versions_attributes, :versions, :name
   accepts_nested_attributes_for :versions
 
+
+  def self.find_with_locid(locid, _iri=nil)
```

Comment:
Style/SpaceAroundEqualsInParameterDefault: Surrounding space missing in default value assignment.